kiliautoml/models/_detectron2_semantic_segmentation.py

```diff
@@ -161,7 +167,7 @@ def train(
         for d in ["train", "val"]:
             # TODO: separate train and test
             DatasetCatalog.register(
-                "dataset_" + d, lambda d=d: self._convert_coco_to_detectron(data_dir)
+                "dataset_" + d, lambda d=d: self._convert_coco_to_detectron(data_dir, mode=d)
```

Comment:
why is there a `d=d` instead of just `d`?

Reply:
https://stackoverflow.com/questions/68867767/what-does-lambda-x-x-mean